From 5ec3df8e98f51e21fe1f46633b6085897f9b040e Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Thu, 12 Aug 2021 10:17:39 +0900
Subject: [PATCH] ksmbd: update the comment for smb2_get_ksmbd_tcon()
Git-commit: 5ec3df8e98f51e21fe1f46633b6085897f9b040e
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Update the comment for smb2_get_ksmbd_tcon().

Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/ksmbd/smb2pdu.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/fs/ksmbd/smb2pdu.c b/fs/ksmbd/smb2pdu.c
index 0de4163978ce..4ac4fe22edde 100644
--- a/fs/ksmbd/smb2pdu.c
+++ b/fs/ksmbd/smb2pdu.c
@@ -85,10 +85,11 @@ struct channel *lookup_chann_list(struct ksmbd_session *sess, struct ksmbd_conn
 }
 
 /**
- * smb2_get_ksmbd_tcon() - get tree connection information for a tree id
+ * smb2_get_ksmbd_tcon() - get tree connection information using a tree id.
  * @work:	smb work
  *
- * Return:      matching tree connection on success, otherwise error
+ * Return:	0 if there is a tree connection matched or these are
+ *		skipable commands, otherwise error
  */
 int smb2_get_ksmbd_tcon(struct ksmbd_work *work)
 {
-- 
2.33.0

